cmd/internal/cli/singularity.go

```diff
@@ -413,6 +413,10 @@ var singularityCmd = &cobra.Command{
 
 // RootCmd returns the root singularity cobra command.
 func RootCmd() *cobra.Command {
+	// We must Init() as loading commands etc. is deferred until this is called.
+	// Using true here will result in local docs including any content for installed
+	// plugins.
+	Init(true)
```

Comment:
@cclerget - moved this now. Note that in the `docs.go` I am calling `Init(true)` as I we may want to have the ability to generate docs that include any plugin related modifications?

In `cmdtree.go` it's still `Init(false)` because that's used for e2e command coverage stuff... which wouldn't cover any plugins etc.

Does that sound fair?

Reply:
@dctrud Sounds good :+1: